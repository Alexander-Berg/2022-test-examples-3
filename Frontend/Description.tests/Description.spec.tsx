import * as React from 'react';
import { shallow } from 'enzyme';
import { DescriptionPresenter } from '../Description';
import { DescriptionMeta } from '../Meta/Description-Meta';

describe('Блок Description', () => {
    it('Соответствует снепшоту', () => {
        const wrapper = shallow(
            <DescriptionPresenter>
                <div className="content">Случайный контент</div>
                <DescriptionMeta>
                    <div className="date">29.10.2020</div>
                    <div className="sociality-controls">stub</div>
                </DescriptionMeta>
            </DescriptionPresenter>
        );

        expect(wrapper).toMatchSnapshot();
    });
});
