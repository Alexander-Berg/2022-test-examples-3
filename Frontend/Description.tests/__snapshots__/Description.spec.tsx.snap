// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Блок Description Соответствует снепшоту 1`] = `
<div
  className="turbo-description unit unit_text_s description"
>
  <div
    className="content"
  >
    Случайный контент
  </div>
  <DescriptionMeta>
    <div
      className="date"
    >
      29.10.2020
    </div>
    <div
      className="sociality-controls"
    >
      stub
    </div>
  </DescriptionMeta>
</div>
`;
